mshadow/half.h

```diff
@@ -10,7 +10,11 @@
 #include "./base.h"
 
 #if MSHADOW_USE_F16C
-  #include <x86intrin.h>
```

Comment:
Users also report that this breaks android build

Reply:
Could you point me to the issue for this if it exists?